From 990e4088a2cd5563e7546a275c422479992b6989 Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sonymobile.com>
Date: Thu, 13 Aug 2015 15:55:28 +0200
Subject: [PATCH 2/2] add support for Qualcomm msm8994

msm8994 uses same audio HAL as msm8974 and we can safely
enable build

Change-Id: I0fb77e06f4613d19ceb4885c9225aaecfdf07d28
Signed-off-by: Alin Jerpelea <alin.jerpelea@sonymobile.com>
---
 Android.mk     | 2 +-
 hal/Android.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 6dbcccd..91bc4f2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,4 +1,4 @@
-ifneq ($(filter msm8960 msm8226 msm8x26 msm8974 msm8x74 msm8x84 msm8084,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8960 msm8226 msm8x26 msm8974 msm8x74 msm8x84 msm8084 msm8994,$(TARGET_BOARD_PLATFORM)),)
 
 MY_LOCAL_PATH := $(call my-dir)
 
diff --git a/hal/Android.mk b/hal/Android.mk
index ccace92..1085566 100644
--- a/hal/Android.mk
+++ b/hal/Android.mk
@@ -7,7 +7,7 @@ include $(CLEAR_VARS)
 LOCAL_ARM_MODE := arm
 
 AUDIO_PLATFORM := $(TARGET_BOARD_PLATFORM)
-ifneq ($(filter msm8974 msm8226 msm8084,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8974 msm8226 msm8084 msm8994,$(TARGET_BOARD_PLATFORM)),)
   # B-family platform uses msm8974 code base
   AUDIO_PLATFORM = msm8974
 ifneq ($(filter msm8226,$(TARGET_BOARD_PLATFORM)),)
-- 
2.8.4 (Apple Git-73)

